packages/devtools_app/integration_test/test_infra/_run_test.dart

```diff
@@ -103,7 +107,12 @@ class TestRunner with IOMixin {
         if (enableExperiments) '--dart-define=enable_experiments=true',
       ],
     );
-    listenToProcessOutput(process);
+    listenToProcessOutput(
+      process,
+      onError: (line) {
+        print('on error, would be throwing exception here: $line');
```

Comment:
nit: I'd make this more concise, e.g.
`$processName: $line` or
`chromeDriver: $line`

Reply:
this print was just for debugging, but will apply this comment if we end up throwing an exception from this callback 👍